CHANGELOG.markdown

```diff
@@ -7,6 +7,8 @@
   when building with `template-haskell-2.18` or later.
 * Define `_CharTyLit` in `Language.Haskell.TH.Lens` when building with
   `template-haskell-2.18` or later.
+* Add prisms for working with text with prefixes or suffices in
+  `Data.Text.Strict.Lens` and `Data.Text.Lazy.Lens`
```

Comment:
This should also mention that `prefixed` and `suffixed` have been removed from `Data.List.Lens` and moved to `Control.Lens.Prism`, as this is a breaking change.